pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[project]
+name = "sktime"
+version = "0.9.0"
+description = "A unified framework for machine learning with time series"
+license = "BSD-3-Clause"
+authors = [
```

Comment:
we probably want to add other names here - CC and/or core devs?
We should add at least the other past and present CC members, @TonyBagnall, @mloning, and @aiwalter.
For maintainers, the current CC, @TonyBagnall and @aiwalter.

Reply:
Would @TonyBagnall @aiwalter and @mloning like their contact emails present as well? If so please provide the information.

Reply:
Reached out to @aiwalter for email but haven't heard back. I'll resolve this for now to be able to merge and we can make the updates later if requested :)